metalware-new/src/commands/repo/update.rb

```diff
@@ -1,13 +1,18 @@
 
+require 'commands/basecommand'
 require 'rugged'
 
 require 'constants'
 
 module Metalware
   module Commands
     module Repo
-      class Update
-	def initialize(_args, options)
+      class Update < BaseCommand
+	      def setup(args, options)
```

Comment:
Yeah the indentation was a bit screw when I opened it. Will have another pass over the files